### PR TITLE
Update authFilter descriptions and samples

### PR DESCRIPTION
**Pull request by utle**

### CODE REVIEW

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -5,49 +5,48 @@
 #NLS_MESSAGEFORMAT_NONE
 
 authFilter=Authentication Filter
-authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers to determine whether or not the HTTP request is selected for the authentication.
+authFilter.desc= Specifies a selection rule that represents conditions that are matched against the HTTP request headers. If all conditions are met, the HTTP request is selected for the authentication.
 
 webApp=Web Application
-webApp.desc=Specifies the web application name.
+webApp.desc=Specifies the web application name. For example, <webApp id="sample" name="app1|app2" matchType="notContain"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:11]: Suggested update:
"Specifies the web application name, for example, `<webApp id="sample" name="app1|app2" matchType="notContain"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -5,49 +5,48 @@
...
 
 requestUrl=Request URL
-requestUrl.desc=Specifies the request URL.
+requestUrl.desc=Specifies the HTTP request URL. For example, <requestUrl id="sample" urlPattern="/account/checking|/account/saving" matchType="contains" />
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:14]: Suggested update:
"Specifies the HTTP request URL, for example, `<requestUrl id="sample" urlPattern="/account/checking|/account/saving" matchType="contains" />`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -5,49 +5,48 @@
...
 
 urlPattern=URL pattern
-urlPattern.desc=Specifies the URL pattern.
-
-remoteAddress=Remote Host
-remoteAddress.desc=Specifies the remote host.
+urlPattern.desc=Specifies the URL pattern. The "*", when used as a wildcard, it is not supported.
+remoteAddress=Remote Address
+remoteAddress.desc=Specifies the remote host TCP/IP address of the client application that sent the HTTP request. For example, <remoteAddress id="sample" ip="100.100.100.99" matchType="greaterThan"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:19]: Suggested update:
"Specifies the remote host TCP/IP address of the client application that sent the HTTP request, for example, `<remoteAddress id="sample" ip="100.100.100.99" matchType="greaterThan"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 4]
```diff
@@ -5,49 +5,48 @@
...
 requestUrl=Request URL
-requestUrl.desc=Specifies the request URL.
+requestUrl.desc=Specifies the HTTP request URL. For example, <requestUrl id="sample" urlPattern="/account/checking|/account/saving" matchType="contains" />
 
 urlPattern=URL pattern
-urlPattern.desc=Specifies the URL pattern.
-
-remoteAddress=Remote Host
-remoteAddress.desc=Specifies the remote host.
+urlPattern.desc=Specifies the URL pattern. The "*", when used as a wildcard, it is not supported.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:17]: I'm not really sure what "it" is referring to in this one. 

Should it be: 
"Specifies the URL pattern. When the `*` character is used as a wildcard, the URL pattern is not supported."

Or maybe:
"Specifies the URL pattern. The `*` character is not supported to be used as a wildcard."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 5]
```diff
@@ -5,49 +5,48 @@
...
+remoteAddress=Remote Address
+remoteAddress.desc=Specifies the remote host TCP/IP address of the client application that sent the HTTP request. For example, <remoteAddress id="sample" ip="100.100.100.99" matchType="greaterThan"/>
 
 ip=IP address
-ip.desc=Specifies the IP address.
+ip.desc=Specifies the remote host TCP/IP address. 
 
 host=Host
-host.desc=Specifies the host name.
+host.desc=Specifies the host name. For example, <host id="sample" name="host.mycomp.com" matchType="contains"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:25]: Suggested update:
"Specifies the hostname, for example, `<host id="sample" name="host.mycomp.com" matchType="contains"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 6]
```diff
@@ -5,49 +5,48 @@
...
 
 name=Name
 name.desc=Specifies the name.
 
 value=Value
-value.desc=Specifies the value.
-valueOptional.desc=The value attribute specifies the HTTP request header value. If the value is not specified, then matching is done with the name attribute, not the value attribute.
+value.desc=Specifies the HTTP request header value.
+valueOptional.desc=The value attribute specifies the request header value. If the value is not specified, then matching is done with the name attribute, not the value attribute. For example, <requestHeader id="sample" name="email" matchType="contains"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:32]: Suggested update:
"The value attribute specifies the value of the request header. If the value is not specified, then the name attribute is used for matching, for example, `<requestHeader id="sample" name="email" matchType="contains"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 7]
```diff
@@ -5,49 +5,48 @@
...
 
 userAgent=User Agent
-userAgent.desc=Specifies the user agent.
+userAgent.desc=Specifies the user agent. For example, <userAgent id="sample" agent="Mozilla/5.0" matchType="contains"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:35]: Suggested update:
"Specifies the user agent, for example, `<userAgent id="sample" agent="Mozilla/5.0" matchType="contains"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 8]
```diff
@@ -5,49 +5,48 @@
...
 
 agent=Agent
-agent.desc=Specifies the user agent
+agent.desc=Specifies the brower's user agent string helps identify which browser is being used. 
 
 requestHeader=Request Header
-requestHeader.desc=Specifies the request header attribute name.
+requestHeader.desc=Specifies the HTTP request header attribute name. For example, <requestHeader id="sample" name="email" value="kevin@yahoo.com|steven@gmail.com" matchType="contains"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:41]: Suggested update:
"Specifies the HTTP request header attribute name, for example, `<requestHeader id="sample" name="email" value="kevin@yahoo.com|steven@gmail.com" matchType="contains"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 9]
```diff
@@ -5,49 +5,48 @@
...
 
 cookie=Cookie
-cookie.desc=Specifies the cookie name.
+cookie.desc=Specifies the cookie name that client send in the HTTP request. <cookie id="sample" name="LtpaToken2" matchType="equals"/>
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:44]: Suggested update:
"Specifies the name of the cookie that the client sends in the HTTP request, for example, `<cookie id="sample" name="LtpaToken2" matchType="equals"/>`."

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties [hunk 10]
```diff
@@ -5,49 +5,48 @@
...
+value.desc=Specifies the HTTP request header value.
+valueOptional.desc=The value attribute specifies the request header value. If the value is not specified, then matching is done with the name attribute, not the value attribute. For example, <requestHeader id="sample" name="email" matchType="contains"/>
 
 userAgent=User Agent
-userAgent.desc=Specifies the user agent.
+userAgent.desc=Specifies the user agent. For example, <userAgent id="sample" agent="Mozilla/5.0" matchType="contains"/>
 
 agent=Agent
-agent.desc=Specifies the user agent
+agent.desc=Specifies the brower's user agent string helps identify which browser is being used.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:38]: Suggested update:
"Specifies the browser's user agent to help identify which browser is being used. "

File: dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties (hunk 4 above)
Comment (utle) [reply dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties:17]: "Specifies the URL pattern. The * character is not supported to be used as a wildcard."

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by utle** [comment]
#run-libby-bot

**Pull request merged**